overlay.go

```diff
@@ -440,6 +440,9 @@ func ReadAndMergeBundleData(sources ...BundleDataSource) (*BundleData, error) {
 		incResolver := sources[srcIndex].ResolveInclude
 		basePath := sources[srcIndex].BasePath()
 		for app, appData := range base.Data.Applications {
+			if appData == nil {
```

Comment:
Should we also bail out if it's empty? Or will that interfere with the short-syntax work?

Reply:
Bingo, I think I can use this for short syntax as well :)